test/spec/bulk-memory-operations/binary.txt

```diff
@@ -1,6 +1,6 @@
 ;;; TOOL: run-interp-spec
 ;;; STDIN_FILE: third_party/testsuite/proposals/bulk-memory-operations/binary.wast
-;;; ARGS*: --enable-bulk-memory
```

Comment:
This seems maybe wrong.. unless bulk memory depends on reference types?   But if it did this flag would not be needed.